packages/plugin/src/Form/Form.php

```diff
@@ -742,24 +736,6 @@ public function jsonSerialize(): array
             ],
         ];
 
-        $reCaptchaEnabled = ReCaptchaHelper::canApplyReCaptcha($this);
-
-        $settingsModel = Freeform::getInstance()->settings->getSettingsModel();
-
-        $isHCaptcha = \in_array($settingsModel->getRecaptchaType(), [SettingsModel::RECAPTCHA_TYPE_H_INVISIBLE, SettingsModel::RECAPTCHA_TYPE_H_CHECKBOX], true);
-
-        if ($reCaptchaEnabled) {
-            $object['reCaptcha'] = [
-                'enabled' => true,
-                'handle' => 'reCaptcha',
-                'name' => $isHCaptcha ? 'h-recaptcha-response' : 'g-recaptcha-response',
-            ];
-        } else {
-            $object['reCaptcha'] = [
-                'enabled' => false,
-            ];
-        }
-
```

Comment:
Lets remember to check this again when we update GraphQL, as this was used in the Form queries...I see no comment to refactor